pages/product/forkedModel.php

```diff
@@ -0,0 +1,57 @@
+<?php
+	echo "<!DOCTYPE html>";
```

Comment:
Can you turn off error_reporting just in case it tries to get the cookie below and it isn't set?